### ISSUE TITLE
Ability to flag "notes" to be hidden or deleted when containing inappropriate text

### ISSUE THREAD

**Issue by tomhughes** [tomhughes/trac-tickets#4915]
**[Submitted to the original trac issue database at 12.41am, Thursday, 25th July 2013]**

There needs to be an ability for registered users to be able to flag a note if it contains inappropriate text, or is completely spam.

The main reason I bring this up is because of this new "note" that was made today:

http://www.openstreetmap.org/browse/note/16991

Something like this needs to be hidden/deleted.  We mappers need some way to flag them so they can be eliminated from the public view.

**Comment by tomhughes** [comment]
**Author: Bryce C Nesbitt**
**[Added to the original trac issue at 5.24pm, Thursday, 26th February 2015]**

Turns out there IS a hidden state for test.
Members of the DWG (data working group) can suppress a given note.

---
I learned of this only because of some statistics work (See http://resultmaps.neis-one.org/osm-notes-overview ).

---
It is only matter of time before Viagra spammers discover the notes system and have a field day.
